test/test_crud.py

```diff
@@ -1,16 +1,50 @@
 import pytest
+import os
+import tempfile
+import json
 from ansible.cli.playbook import PlaybookCLI
 
 
 MODULES = ['organization', 'product']
 
 
-def run_playbook(path, extra_vars=""):
-    cli = PlaybookCLI(['ansible-playbook', path, "--extra-vars", extra_vars])
+def run_playbook_vcr(module, extra_vars=None, extra_args=None, record=False):
+    # Assemble extra parameters for playbook call
+    if extra_args is None:
+        extra_args = []
+    extra_args.extend(['--inventory', 'test/inventory/hosts'])
+    if record:
+        # Cassettes that are to be overwritten must be deleted first
+        record_mode = 'once'
+    else:
+        # Never reach out to the internet
+        record_mode = 'none'
+        # Only run the tests (skip fixtures)
+        extra_args.extend(['--limit', 'tests'])
+
+    # Dump recording parameters to json-file and pass its name by environment
+    test_params = {'test_name': module, 'serial': 0, 'record_mode': record_mode}
+    with tempfile.NamedTemporaryFile('w', suffix='json', prefix='fam-vcr-') as params_file:
+        json.dump(test_params, params_file.file)
+        params_file.file.close()
+        os.environ['FAM_TEST_VCR_PARAMS_FILE'] = params_file.name
+        return run_playbook(module, extra_args=extra_args, extra_vars=extra_vars)
```

Comment:
nitpick: args and vars are sorted the other way round in the `run_playbook` signature, so you might want to have them properly here too (even so it does not matter as you call them named).

Reply:
Will adjust that ;)